README.md

```diff
@@ -5,7 +5,7 @@ TinyFont
 
 TinyFont is a font/text package for [TinyGo](https://tinygo.org/) displays. It is heavily based on [Adafruit's GFX library](https://github.com/adafruit/Adafruit-GFX-Library).
 
-![example](https://raw.githubusercontent.com/conejoninja/tinyfont/master/example.png)
+![example](https://raw.githubusercontent.com/conejoninja/tinyfont/master/tinyfont.png)
```

Comment:
This image should be pointed to the tinygo-org organization as well.

Reply:
@deadprogram 
Oh.
I've fixed the URL.